fixtures/hasura/chinook/metadata/models/Customer.hml

```diff
@@ -7,31 +7,31 @@ definition:
     - name: id
       type: Chinook_ObjectId!
     - name: address
-      type: String!
```

Comment:
Lots of types have been changed from non-nullable to nullable which I think should not have happened. I want to investigate that.

Reply:
We changed the default, didn't we?

Reply:
I have `allSchemaNullable: false` so I'd expect non-nullable types. But I did file a ticket because introspection on the sample_mflix connector changes the setting to `true` instead of leaving it as-is. That hasn't happened for the chinook connector configuration. I don't know what the difference is. https://hasurahq.atlassian.net/browse/MDB-172

Reply:
Never mind about that bug - it was user error.

To get the configuration I wanted to stick I needed to put a newer modification time on the configuration file, and I needed to make it valid JSON. The things I tried were to revert the change using git, which I think rolled back the modification time, and when I edited by hand I put in a trailing comma which was not valid JSON.

Reply:
Never mind, it looks like these nullable changes are also expected behavior. I was too tired when I was looking over this yesterday evening